src/libsync/owncloudpropagator.cpp

```diff
@@ -213,24 +209,13 @@ static void blacklistUpdate(SyncJournalDb *journal, SyncFileItem &item)
     // Suppress the error if it was and continues to be blacklisted.
     // An ignoreDuration of 0 mean we're tracking the error, but not actively
     // suppressing it.
-    if (item._hasBlacklistEntry && newEntry._ignoreDuration > 0) {
+    if (item._status != SyncFileItem::SoftError && item._hasBlacklistEntry && newEntry._ignoreDuration > 0) {
         item._status = SyncFileItem::BlacklistedError;
-
-        qCInfo(lcPropagator) << "blacklisting " << item._file
-                             << " for " << newEntry._ignoreDuration
-                             << ", retry count " << newEntry._retryCount;
-
-        return;
     }
 
-    // Some soft errors might become louder on repeat occurrence
-    if (item._status == SyncFileItem::SoftError
-        && newEntry._retryCount > 1) {
-        qCWarning(lcPropagator) << "escalating soft error on " << item._file
-                                << " to normal error, " << item._httpErrorCode;
-        item._status = SyncFileItem::NormalError;
-        return;
-    }
```

Comment:
Hmm, this is going away completely? Which soft errors were ending up here, that were escalated to NormalErrors before?

Reply:
Hmm maybe I should look for _httpErrorCode that way http errors will behave as before.